docs/releases/2.1.txt

```diff
@@ -391,6 +391,10 @@ Miscellaneous
 * Management commands no longer allow the abbreviated forms of the
   ``--settings`` and ``--pythonpath`` arguments.
 
+* The private ``django.db.models.sql.constants.QUERY_TERMS`` constant is
+  removed. Use the ``get_lookup()``/``get_lookups()`` methods of the
```

Comment:
Use links for get_lookup/s? I'd phrase it that those "may be suitable alternatives".